site/docs/v1/tech/migration-guide/supabase.adoc

```diff
@@ -0,0 +1,299 @@
+---
+layout: doc
+title: Migration From Supabase
+description: How to migrate your users to FusionAuth from Supabase.
+---
+:page-liquid:
+
+:migration_source_name: Supabase
+:migration_source_dir: supabase
+:extra_permissions_text:
+
+
+== Overview
+
+This document will help you migrate off of Supabase and onto FusionAuth.
+
+This guide assumes you have installed FusionAuth. If you have not, please link:/docs/v1/tech/installation-guide/[view our installation guides] and install FusionAuth before you begin. For more general migration information, please view the link:/docs/v1/tech/migration-guide/general[FusionAuth migration guide].
+
+* <<Planning Considerations>>
+** <<Obtaining User Data>>
+** <<Mapping User Attributes>>
+** <<Social Logins>>
+** <<Other Entities>>
+* <<Exporting Users>>
+* <<Importing Users>>
+** <<Set Up FusionAuth>>
+** <<Get the Script>>
+** <<Install Needed Gems>>
+** <<Use the Script>>
+** <<Verify the Import>>
+** <<The Final Destination of Imported Users>>
+* <<What to Do Next>>
+* <<Additional Support>>
+
+
+There are a number of different ways applications can be integrated with Supabase, and it would be difficult to cover them all. This guide mentions the typical parts of a bulk migration and focuses on migrating user data from a Supabase-managed user database into FusionAuth.
+
+== Planning Considerations
+
+=== Obtaining User Data
+
+You can write a custom psql script in the Supabase SQL Editor to export user data as JSON that you can import to FusionAuth. In Supabase, user data is stored in the `auth.users` table, and you can join the query with data from other `auth` tables to pull specific attributes.
+
+=== Mapping User Attributes
+
+include::docs/v1/tech/migration-guide/_mapping-user-attributes.adoc[]
+
+
+=== Social Logins
+
+include::docs/v1/tech/migration-guide/_social-login-note.adoc[]
+
+include::docs/v1/tech/migration-guide/_social-login-migration.adoc[]
+
+=== Other Entities
+
+:other_migrated_entities: connections or roles
+include::docs/v1/tech/migration-guide/_other-entities-intro.adoc[]
+:other_migrated_entities!:
+
+* In Supabase, sign-in providers are a source of data for users. FusionAuth calls these Identity Providers.
+* Projects are a high-level construct that groups entities such as users and applications together. FusionAuth calls these Tenants.
+* Supabase has other products your applications might be using, like Edge functions. FusionAuth does not have equivalents for these products, and is focused only on authentication.
+
+==== Identifiers
+
+When creating an object with the FusionAuth API, you can specify the Id. It must be a link:/docs/v1/tech/reference/data-types#uuids[UUID].
+
+This works for users, applications, and tenants, among others.
+
+Supabase uses a custom UUID format. When migrating, a new UUID will be created for the user on FusionAuth, or you can use the Supabase user Id migration.
```

Comment:
What is the Supabase user Id migration? I am not familiar with that.

Reply:
That was an error. I have removed that bit.